projects/go-lib/src/public_api.ts

```diff
@@ -18,6 +18,9 @@ export * from './lib/components/go-action-sheet/go-action-sheet.module';
 export * from './lib/components/go-badge/go-badge.component';
 export * from './lib/components/go-badge/go-badge.module';
 
+// Branding
+export * from './lib/go-config.service';
```

Comment:
Should we also export the `GoConfigInterface` here?

Reply:
Yes we should